tidb-specific-system-variables.md

```diff
@@ -736,3 +736,13 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
 默认值：0
 
 这个变量用来显示上一个 `execute` 语句所使用的执行计划是不是直接从 plan cache 中取出来的。
+
+### tidb_allow_batch_cop <span class="version-mark">从 v4.0 版本开始引入</span>
```

Comment:
This change will be overwritten by docs-special-week changes. Please check the latest version of this file.

Reply:
The latest doc is https://pingcap.com/docs-cn/dev/tidb-specific-system-variables/#tidb_allow_batch_cop-%E4%BB%8E-v40-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5

Reply:
Confirmed with @JaySon-Huang : will keep the change of https://github.com/pingcap/docs-cn/pull/3159/files#diff-36f1f55efb0294795b77879ecad344a0R869